### ISSUE TITLE
Annual Report modals should return focus to CTA on modal close

### ISSUE THREAD

**Issue by alexgibson** [mozilla/bedrock#12346]
## Description

When closing modal on the annual report pages, focus is lost.

### Steps to reproduce

1. Open https://www.mozilla.org/en-US/foundation/annualreport/2020/
2. Click a card to open a modal.
3. Close the modal by clicking 'X'

### Expected result

Focus should be returned to the opening element / link.

### Actual result

Focus is lost (goes back to the `<html>` element.

It looks like this is happening because some of the clickable elements are `<section>` elements, which do not receive `focus()` naturally.

### Environment

Firefox Nightly
Chrome
Safari

**Comment by alexgibson** [comment]
Fixed in https://github.com/mozilla/bedrock/pull/12384
